packages/components/src/popover/test/index.js

```diff
@@ -2,110 +2,83 @@
  * External dependencies
  */
 import TestUtils from 'react-dom/test-utils';
-import ReactDOM from 'react-dom';
-import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Popover from '../';
 
-describe( 'Popover', () => {
-	describe( '#componentDidUpdate()', () => {
-		let wrapper;
-		beforeEach( () => {
-			jest.spyOn( Popover.prototype, 'computePopoverPosition' ).mockImplementation( noop );
-			jest.spyOn( Popover.prototype, 'toggleAutoRefresh' ).mockImplementation( noop );
-		} );
+jest.useFakeTimers();
 
-		afterEach( () => {
-			jest.restoreAllMocks();
+class PopoverWrapper extends Component {
```

Comment:
TestUtils don't work with functional components. I think it's the same reason the dropdown asserters were failing.

Reply:
How about using this new util for hooks? I remembered they had introduced something:
https://reactjs.org/docs/hooks-faq.html#how-to-test-components-that-use-hooks

Reply:
Which util? We're using this one already

Reply:
I think @gziolo is referring to https://reactjs.org/docs/test-utils.html#act

Reply:
Right! I'm using it here https://github.com/WordPress/gutenberg/pull/15053/files#diff-829fd588eade3365fbb33ef908ef9fe8R41

Reply:
Sidenote:  Via the docs for the above, there is mention of [this library](https://github.com/kentcdodds/react-testing-library).  Looks kind of interesting.

Reply:
> Sidenote: Via the docs for the above, there is mention of [this library](https://github.com/kentcdodds/react-testing-library). Looks kind of interesting.

We still have over 80 tests using Enzyme 🤷‍♂️We can investigate, but all those abstractions seem to bring

Reply:
Ya I agree.  I only referenced the other library because it supports the latest in React and on the surface it looks like they're doing a much better job than enzyme of keeping up to date.  I'll probably experiment with the library in my own projects for a bit.